marathon-cli.py

```diff
@@ -103,13 +117,15 @@ def print_file_chunk(url, offset, auth):
     mesos_agent_map_string = os.getenv("MESOS_AGENT_MAP", None)
     mesos_master_urls = os.getenv("MESOS_MASTER_URLS", "http://localhost:5050").split(',')
 
+    pp = pprint.PrettyPrinter(depth=2)
```

Comment:
I would move this line down to the paragraph where the loggers are initialized with a comment as to its use